lib/publiccloud/instance.pm

```diff
@@ -580,7 +580,9 @@ Set PUBLIC_CLOUD_PERF_COLLECT true or >0, to activate boottime measurements.
 
 sub measure_boottime() {
     my ($self, $instance, $type) = @_;
-    return 0 unless (get_var('PUBLIC_CLOUD_PERF_COLLECT'));
+    my $data_collect = get_var('PUBLIC_CLOUD_PERF_COLLECT', 1);
```

Comment:
```suggestion
    return 0 unless (get_var('PUBLIC_CLOUD_PERF_COLLECT', 1));
```

It's a style question but I'd argue this is trivial enough that we don't need an additional variable.

Reply:
I expected this effect, but I preferred to leave more visible the parameter handling by separating the value assignment from usage, helpful in case of explicit FALSE/0 assignments to disable.

Reply:
> It's a style question but I'd argue this is trivial enough that we don't need an additional variable.
by removing variable you removing ability to disable this flow . While I want it to be default behavior I still would like to have ability to switch it off in some certain cases

Reply:
I don't understand, but as this is just a style suggestion, just ignore it.

Reply:
arrrgh ! :roll_eyes:  :see_no_evil:  :speak_no_evil: . and only NOW I realized what Felix was suggesting :D :D :D Felix the problem that we have TWO variables here one is perl variable and another is openQA variable . Maybe it was obvious to you that you referring to perl one but at least me get that you asking to remove **openQA one**  and of course I totally against this . But about removing **perl variable** it make sense yeah +1 from my side . 

Not sure what was Maurizio point :thinking:  Maurizio did you get into same trap as I do ?

Reply:
Please don't bother. This is just a style suggestion, not worth discussion it for more than 10 seconds.

Just leave it as it is (or change it), it is just a suggestion.

Reply:
I got that it was a style preference and sorry for keeping my position because this is my style, also because with or without variable in case of exit the timing is not so different, but 'with' it, the code legibility is improved and is less spagetti-style :-)